Makefile

```diff
@@ -243,30 +244,43 @@ project/nmdc_schema_merged.yaml:
 #		--output project/prefixmap/nmdc.json \
 #	  	--mergeimports $@
 
-project/nmdc_materialized_patterns.yaml:
+#project/nmdc_materialized_patterns.yaml:
+#	$(RUN) gen-linkml \
```

Comment:
Is there reason to comment out & not just remove?

Reply:
no, I should have deleted instead of commenting-out. I will do that in a  subsequent PR.